### DISCUSSION TITLE
1000 units of T41U5XBB

### DISCUSSION THREAD

**Discussion by phil-barrett** [grblHAL/core#268] [General]
Today, I shipped the 1000th T41U5XBB - Teensy 4.1 5 Axis Breakout Board for grblHAL.  Given the amount of soldering it takes (32 screw terminals, 11 pinheaders, 2 strip sockets and a trimpot), I am surprised it has done that well. Yet, the power of the Teensy 4.1 and richness of grblHAL have made it highly desirable among system builders, enough to overcome the hurdle of soldering.

I lift my drink to Terje!  Without you, it would never have happened. Through your quiet, steadfast and tireless efforts, grblHAL has inherited the mantle of leadership in the Grbl world. In addition, grblHAL is being recognized as serious contender in the CNC world. You had a clear roadmap 5 years ago and it has proven to be the right one.  Thank you for all your efforts.

Phil

**Comment by terjeio** [comment]
Great news!!
And many thanks to you as well - without people like you providing high quality hardware grblHAL would have moved nowhere.

**Reply by phil-barrett** [reply]
Thanks! Hopefully, even better things are in the future.
